### PR TITLE
[Js] Pop-up (typeFood page)

### LINKED ISSUE #45: [Js] Pop-up  (typeFood page)

### PR DESCRIPTION
**Pull request by farahalzard**
Relates #45

### CODE REVIEW

**Review by IsraaHamdi — APPROVED** [review]

File: js/typeFood.js [hunk 1]
```diff
@@ -4,21 +4,24 @@ const searchInp = document.querySelector(".search-control");
 const holder = document.querySelector(".cards-food");
 let titelCat = document.querySelector(".header-search");
 let catName = localStorage.getItem('catFood');
+let parent = document.querySelector(".overlay");
```
Comment (yaqootturman) [inline js/typeFood.js:7]: parent is null

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: js/typeFood.js (hunk 1 above)
Comment (yaqootturman) [inline js/typeFood.js:7]: parent is null

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: js/typeFood.js (hunk 1 above)
Comment (farahalzard) [reply js/typeFood.js:7]: its functionality to close and open the popup

Comment (yaqootturman) [reply js/typeFood.js:7]: yes, and you cant add style for null, so that makes an error
check it before replying,please!

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: js/typeFood.js (hunk 1 above)
Comment (farahalzard) [reply js/typeFood.js:7]: I already checked it, it works.

**Review by muhannaAtalla20 — APPROVED** [review]

**Review by yaqootturman — APPROVED** [review]

**Comment by farahalzard** [comment]
#17

**Comment by farahalzard** [comment]
#19 #20 #21

**Comment by farahalzard** [comment]
#19 #20 #21

**Pull request merged**